client/src/assets/components/posts/PostBody.jsx

```diff
@@ -1,26 +1,34 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import ReactMarkdown from 'react-markdown';
 
 const PostBody = ({ body }) => {
     const [expandedPost, setExpandedPost] = useState(false);
 
+    // const renderBodyWithTags = (text) => {
+    //     return text.split(' ').map((word, i) => {
+    //         if (word.startsWith('#')) {
+    //             const cleanTag = word.replace(/[^\w#]/g, '').slice(1).toLowerCase();
+    //             return (
+    //                 <Link
+    //                     key={i}
+    //                     // to={`/tags/${cleanTag}`}
+    //                     to={`#`}
+    //                     className="text-primary hover:underline"
+    //                 >
+    //                     {word + ' '}
+    //                 </Link>
+    //             );
+    //         }
+    //         return <span key={i}>{word + ' '}</span>;
+    //     });
+    // };
+    
```

Comment:
Since it seems to be functional, let's go ahead and remove this and carry on with your updates. 😄